mapie/tests/test_classification.py

```diff
@@ -681,7 +681,7 @@ def test_sum_proba_to_one_fit(y_pred_proba: ArrayLike) -> None:
     sum to one, return an error in the fit method.
     """
     wrong_model = WrongOutputModel(y_pred_proba)
-    mapie_clf = MapieClassifier(wrong_model)
+    mapie_clf = MapieClassifier(wrong_model, cv="prefit")
```

Comment:
Shouldn't we generalize this test to cross-conformal ?

Reply:
Using cross-conformal, `MapieClassifier` a cross-validation on a two- or one-instance dataset in `Y_PRED_PROBA_WRONG` and tries to adjust the number of labels with a custom `WrongOutputModel` class. I suggest to leave `cv="prefit"` for this test since we just want to test the `_check_proba_normalized` function.